Core/Base/ExtendedController/WidgetButton.php

```diff
@@ -180,26 +180,26 @@ private function getCalculateHTML($label, $value, $hint)
      * Returns the HTML code to display an action button
      *
      * @param string $label
-     * @param string $indexView
      * @param string $hint
+     * @param string $formName
+     * @param string $class
      * @return string
      */
-    private function getActionHTML($label, $indexView, $hint, $class = 'col-sm-auto')
+    private function getActionHTML($label, $hint, $formName = 'main_form', $class = 'col-sm-auto')
     {
+<<<<<<< HEAD
```

Comment:
Aquí se ha colado un merge mal resuelto.

Reply:
Now it should be correct.